src/infrastructure/validation/logger/LoggerFactory.java

```diff
@@ -14,9 +13,6 @@
 	 */
 	private static ILogger loggerManager;
 	
-	/**
-	 * 
-	 */
 	private LoggerFactory() {
 		// TODO Auto-generated constructor stub
 	}
```

Comment:
Can explain why this is private, so that people know that loggerManager can be instantiated only through the getLoggerInstance() call.

Reply:
It's because of the Factory Pattern. I'll add clarification:
> LoggerFactory class provides the method getLoggerInstance(). The approach followed is the factory pattern. This method will return a singleton reference to the object, that implements ILogger interface and can so, be used for logging

Reply:
Is there any way to shorten this since we had this explanation in the file description. Trying to keep comments between code short. If it is not possible you can use the same.